tests/test_app/tests.py

```diff
@@ -118,15 +115,6 @@ def test_get_fields(self):
         fields = self.serializer.get_fields()
         assert isinstance(fields, dict)
 
-    def test_validate_success(self):
-        attrs = {
-            'username': 'test_user',
-            'password': 'test_password',
-        }
-        self.serializer.backend_method_name = ""
-        with pytest.raises(AuthenticationFailed):
-            self.serializer.validate(attrs)
-
```

Comment:
I think a test to check what happens if a backend_method_name is not set, is required.